### PR TITLE
Pack xacml mediator feature

### PR DESCRIPTION
**Pull request by Nirothipan**

### CODE REVIEW

File: components/mediation/mediators/entitlement-mediator/org.wso2.micro.integrator.identity.entitlement.mediator/pom.xml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+                            !org.osgi.framework,
+                            !javax.servlet,
+                            !javax.servlet.http,
+                            !org.apache.axis2.databinding,
+                            !org.apache.axis2.databinding.utils.reader,
+                            !org.pache.axis2.databinding.utils.writer,
+                            !org.wso2.carbon.identity.entitlement.mediator.*,
+                            org.wso2.carbon.identity.entitlement.proxy.*,
+                            org.wso2.carbon.identity.entitlement.stub.*;version="${carbon.identity.imp.pkg.version}",
+                            org.wso2.carbon.authenticator.stub.*;version="${carbon.kernel.imp.pkg.version}",
```
Comment (milindaperera) [inline components/mediation/mediators/entitlement-mediator/org.wso2.micro.integrator.identity.entitlement.mediator/pom.xml]: Are we packing org.wso2.carbon.identity packages?

File: features/mediation-features/entitlement-mediator-feature/org.wso2.micro.integrator.identity.xacml.mediator/pom.xml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+                            <propertiesFile>../../../etc/feature.properties</propertiesFile>
+                            <adviceFile>
+                                <properties>
+                                    <propertyDef>org.wso2.carbon.p2.category.type:server</propertyDef>
+                                </properties>
+                            </adviceFile>
+                            <bundles>
+                                <bundleDef>libthrift.wso2:libthrift</bundleDef>
+                                <bundleDef>org.wso2.ei:org.wso2.micro.integrator.identity.entitlement.mediator
+                                </bundleDef>
```
Comment (milindaperera) [inline features/mediation-features/entitlement-mediator-feature/org.wso2.micro.integrator.identity.xacml.mediator/pom.xml]: Let's format

Comment (Nirothipan) [reply features/mediation-features/entitlement-mediator-feature/org.wso2.micro.integrator.identity.xacml.mediator/pom.xml]: This is formatted , as it exceeds 120 characters, it is in second line.

File: components/mediation/mediators/entitlement-mediator/org.wso2.micro.integrator.identity.entitlement.mediator/pom.xml (hunk 1 above)
Comment (Nirothipan) [reply components/mediation/mediators/entitlement-mediator/org.wso2.micro.integrator.identity.entitlement.mediator/pom.xml]: yes , these were previously packed in mediation, now moved here.

**Pull request merged**